doc/Manuals/ig-cs_x-road_6_central_server_installation_guide.md

```diff
@@ -466,6 +468,58 @@ The interrupted installation can be finished using
 
 `sudo apt-get -f install`
 
+### 5.5 This package can be upgraded up to version 7.x.x
+
+It's possible that the following error message comes up during the central server upgrade.
+
+`This package can be upgraded up to version 7.x.x`
+
+The central server can only be upgraded one or two minor versions in a single step, otherwise the above error occurs. The fix is to upgrade the central server to the target version step by step.
+
+For example, the following central server packages are currently installed.
+
+```
+root@test-cs:~# dpkg -l | grep xroad
+ii  xroad-base                      7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 amd64        X-Road base components
+ii  xroad-center                    7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 all          X-Road central server
+ii  xroad-centralserver             7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 all          X-Road central server
+ii  xroad-centralserver-monitoring  7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 all          Monitoring client configuration for X-Road central
+ii  xroad-confclient                7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 amd64        X-Road configuration client components
+ii  xroad-database-local            7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 all          Meta-package for X-Road local database dependencies
+ii  xroad-jetty9                    7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 all          Jetty9 for X-Road purposes
+ii  xroad-nginx                     7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 amd64        X-Road nginx component
+ii  xroad-signer                    7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 amd64        X-Road signer component
```

Comment:
The development revisions are not used in released packages, the example would be clearer with release version numbers (e.g. 7.0.0-1)

Reply:
Fixed in https://github.com/nordic-institute/X-Road/pull/957/commits/1a68404ce820ee38ef4dfb67d09a108a71a8f74c